CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## 0.6.0-dev
+## 0.7.0-dev
+
+## 0.6.0
```

Comment:
This change is mostly just a suggestion to simplify our release process. That way there should be less commits on master that could be confused with a release.

Reply:
Well, if we add cherry picks, we will have the same amount of commits on master?